src/compose/ComposeBox.js

```diff
@@ -199,10 +199,6 @@ const ComposeBox: React$AbstractComponent<Props, ImperativeHandle> = forwardRef(
 
   const inputBlurTimeoutId = useRef<?TimeoutID>(null);
 
-  // TODO(#5141): Encapsulate this in a nice, plain action-sheet pattern
-  //   instead of setting it all over the place
```

Comment:
This gives us a reminder that we should close #5141 — basically it's an alternative design that's mutually exclusive with #5080.

Reply:
Sure; I can remember to do that after this is merged.